client/my-sites/media-library/header.jsx

```diff
@@ -54,23 +41,49 @@ export default React.createClass( {
 		this.setState( {
 			moreOptionsContext: component
 		} );
-	},
+	}
 
 	toggleAddViaUrl( state ) {
 		this.setState( {
 			addingViaUrl: state,
 			isMoreOptionsVisible: false
 		} );
-	},
+	}
 
 	toggleMoreOptions( state ) {
 		this.setState( {
 			isMoreOptionsVisible: state
 		} );
-	},
+	}
+
+	onClickGoogle() {
+		this.props.onSourceChange( 'google_photos' );
+	}
+
+	getPopoverButtons() {
+		const { translate } = this.props;
+
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
+		const buttons = [
+			<PopoverMenuItem onClick={ this.handleAddUrlOpen } key={ 0 }>
+				{ translate( 'Add via URL', { context: 'Media upload' } ) }
+			</PopoverMenuItem>
+		];
+
+		if ( this.props.onSourceChange ) {
+			buttons.push(
+				<PopoverMenuItem onClick={ this.handleGoogle } key={ 1 }>
+					{ translate( 'Add from Google', { context: 'Media upload' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Add from Google' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.